package.json

```diff
@@ -50,11 +51,16 @@
     "eslint-plugin-jsdoc": "^50.6.9",
     "eslint-plugin-prettier": "^5.2.6",
     "globals": "^15.14.0",
+    "husky": "^9.1.7",
+    "lint-staged": "^15.5.1",
     "livereload": "^0.9.3",
```

Comment:
It looks like husky and lint-staged have been added, which is great for enforcing code quality! However, the husky configuration for the pre-commit hook seems to be missing. To ensure lint-staged runs automatically on staged files during commits, you can include the following configuration in your package.json:

```
"husky": {
  "hooks": {
    "pre-commit": "lint-staged"
  }
}
```
This will trigger lint-staged whenever a commit is made, ensuring that only properly linted code is committed. Let me know if you need help setting this up!

Reply:
Now resolved. This was a local VSCode configuration rather than the code itself.